cellfinder_core/classify/classify.py

```diff
@@ -2,6 +2,7 @@
 
 import numpy as np
 from imlib.general.system import get_num_processes
+from tensorflow import keras
```

Comment:
Just wanted to check that importing keras here no longer prints a load of warnings whenever `cellfinder-core` is imported. This used to happen, and was v. annoying (which is why some of the imports are in functions elsewhere).

Reply:
It doesn't print any warnings, at least for me. This is used for the custom callback class at the bottom of this file. If importing tensorflow here is a worry, I could move the class definition inside the main function? (but that wouldn't as nice code in terms of readbility)

Reply:
If we can have the imports at the top of the file, we should do that. If you haven't seen any issues with it, leave it as it is.